src/Analysis/Ast/Impl/Caching/CacheFolders.cs

```diff
@@ -34,7 +34,7 @@ internal sealed class CacheFolderService: ICacheFolderService {
             // File name depends on the content so we can distinguish between different versions.
             using (var hash = SHA256.Create()) {
                 return Convert
-                    .ToBase64String(hash.ComputeHash(new UTF8Encoding(false).GetBytes(content)))
+                    .ToBase64String(hash.ComputeHash(new UTF8Encoding(encoderShouldEmitUTF8Identifier: false).GetBytes(content)))
```

Comment:
It's not in our convention to require named parameters to include their names. Stuff like this looks kinda silly... The UTF8Encoding docs themselves don't do this.

Reply:
sure. I was looking code and when I saw "false", I wasn't sure what "false" do with UTF8Encoding. so added the named argument. if people prefere not having named argument. I am fine.

Reply:
Yeah, this one's unfortunate. There really should have been a static member of that class which encoded this behavior rather than this weird `false` everywhere to get actual UTF-8 behavior.

Reply:
Yeah, if anyone wants them, there are VS editor extensions that display them from signatures as editor adornments.

Reply:
They make sense sometimes, typically for booleans when it is unclear what true/false might mean.

Reply:
so remove named arguments then?

Reply:
by the way, caching SHA256 and using incremental hash will improve perf quite a bit if it is used a lot. 

Roslyn started same as what it has now but changed to this - http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/Solution/Checksum_Factory.cs,21

made about 3-4x improvement on hashing perf. Roslyn creates SHA for everything in solution, (for Roslyn solution itself more than 40000 SHA), and we saw big improvement on total time we spend on SHA.

probably only matter only if we also does many SHA on big stream. otherwise, probably won't matter much.

Reply:
I am OK with named args for booleans when it is not obvious what it is. Leave it.